Makefile

```
@@ -0,0 +1,120 @@
+.PHONY: help test test-coverage test-be test-fe coverage-report coverage-html clean
+
```

We need to load the environment variables before running tests. Otherwise they fail. With the following statement `make test-be` passes on my local 👍 

```suggestion
ifneq (,$(wildcard ./.env))
    include .env
    export
endif
```